src/core/errorHandler.js

```diff
@@ -64,7 +64,7 @@ export type ErrorHandlerType = typeof ErrorHandler;
  * The decorator will assign an ErrorHandler instance to the errorHandler
  * property.
  *
- * For convenience, you can use withErrorHandling() instead which will
+ * For convenience, you can use withRenderedErrorHandler() instead which will
```

Comment:
This is unrelated but that comment is probably a bit too verbose. I think it would be fine if it just said:

```js
* For convenience, you can use `withRenderedErrorHandler()` which renders
* the error automatically at the beginning of the component's output.
```

thoughts?